### DISCUSSION TITLE
支持 M2M-100 模型

### DISCUSSION THREAD

**Discussion by EFLKumo** [immersive-translate/immersive-translate#1373] [Ideas]
### 新功能 | New Feature

参考 https://developers.cloudflare.com/workers-ai/models/translation/  
通过设置 Cloudflare Account ID 和 API KEY 使用 M2M-100 模型进行翻译

### 为什么您想要添加这些功能 | Why you want to add these features

几乎零成本的 AI 翻译解决方案

**Comment by theowenyoung** [comment]
试试这个项目？ https://github.com/hidenull/translate-cloudflare-ai

**Comment by EFLKumo** [comment]
很好
